packages/cli/src/commands/setup/auth/templates/auth.ts.template

```diff
@@ -29,8 +29,8 @@ type RedwoodUser = Record<string, unknown> & { roles?: string[] }
  */
 export const getCurrentUser = async (
   decoded,
-  { token, type },
-  { event, context }
+  { _token, _type },
```

Comment:
Isn't it better that we just have a eslint comment here instead of doing this?

Reply:
Added 

```
  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

comments